docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
+
+  {"text": "this is an example transcript"}
 
-oracle_vad_manifest [Optional but suggested for good performace. This reduces FPR and TPR]:
-  - To perform just oracle diarization, that is taking speech activity time stamps from groundtruth RTTMs or another VAD from wild instead of NeMo VAD output, ``diarizer`` expects an oracle manifest json file that contains paths to audio files with offset for start time and duration of segment.
+``uem_filepath`` (Optional):
+    UEM file is used for specifying the scoring regions to be evaluated in the given audio file.
+	UEMfile follows the following convention: ``<uniq-id> <channel ID> <start time> <end time>``
+	<channel ID> is set to 1.
 
-To prepare an oracle manifest file, use the script from ``scripts`` folder as shown below:
+    Example line of UEM file:
 
 .. code-block:: bash
+  
+    TS3012d.Mix-Headset 1 12.31 108.98
+    TS3012d.Mix-Headset 1 214.00 857.09
 
-  python $NeMo/scripts/ --paths2rttm_files=<path2groundtruth_rttm_files.txt> --paths2audio_files=<paths2audio_files.txt> --manifest_file=<output_oracle_manifest_file.json>
+``ctm_filepath`` (Optional):
+    
+    CTM file is used for the evaluation of word-level diarization result and word-timestamp alignment.
+	CTM file follows the following convention: ``<uniq-id> <speaker ID> <word start time> <word end time> <word> <confidence>``
+	Since confidence is not required for evaluating diarization results, it can have any value. Note that the ``<speaker_id>`` should be exactly matched with speaker IDs in RTTM. 
 
-Here ``paths2audio_files`` and ``path2groundtruth_rttm_files`` are files containing paths to audio files as shown above.
+    Example lines of CTM file:
 
-AMI Meeting Corpus
-------------------
+.. code-block:: bash
+  
+   TS3012d.Mix-Headset MTD046ID 12.879 0.32 okay 0
+   TS3012d.Mix-Headset MTD046ID 13.203 0.24 yeah 0
 
-The following are the suggested parameters for getting Speaker Error Rate (SER) of 2.13% on AMI Lapel test set corpus:
-  - diarizer.oracle_num_speakers = null (performing unknown speaker case)
-  - diarizer.speaker_embeddings.model_path = ``ecapa_tdnn`` (This model is trained on voxceleb dataset. ``Use this model for simialr non-telephonic speech datasets``)
-  - diarizer.speaker_embeddings.window_length_in_sec = 1.5
-  - diarizer.speaker_embeddings.shift_length_in_sec = 0.75 
 
-Input paths2audio_files, paths2rttm_files and oracle_vad_manifest by following steps as shown above
+Evaluation on benchmark dataset
+-------------------------------
 
-CallHome LDC97S42 (CH109)
--------------------------
+The following instructions can help you reproduce the expected diarization performance on two English dialogue datasets. The following results are evaluations based on 0.25 second collar without evaluating overlapped speech. The evaluation is based on oracle VAD results from RTTM files. Therefore, speaker error rate (SER) is equal to confusion error since oracle VAD has no miss detection or false alarm.
```

Comment:
can help one to reproduce

Reply:
This isn't grammatical error, it is about choice of the word. Fixed